docs/application/native/guides/multimedia/audio.md

```diff
@@ -1,18 +1,21 @@
 # Audio Management
 
 
-You can control the audio behavior of the application. You can control the output volume and sound session types, and query various information from sound devices. You can also use OpenAL to specify the objects and operations in producing high-quality audio output.
+You can control the audio behavior of the application. You can control the output volume and sound conflict scenarios. You can also query various information from sound devices. You can also use OpenAL to specify the objects and operations in producing low-latency audio output.
```

Comment:
You had better change to "You can <strike>also</strike> query various information from sound devices." because the last sentence is starting "You can also ...".